yb-voyager/src/srcdb/yugabytedb_test.go

```diff
@@ -696,3 +696,300 @@ func TestYugabyteGetColumnsWithSupportedTypes_AllScenarios(t *testing.T) {
 		assert.Equal(t, 0, len(unsupportedCols.Keys()), "Expected empty unsupported map for offline migration")
 	})
 }
+
+// TestYugabyteFilterUnsupportedTables tests:
+// 1. Tables with arrays of composite types (UDTs) are always unsupported
+// 2. Tables with arrays of enums are unsupported only with YBGrpcConnector and supported with Logical connector
+// 3. Tables with regular arrays (int[], text[], etc.) are supported
+func TestYugabyteFilterUnsupportedTables_BasicScenarios(t *testing.T) {
```

Comment:
one case if a table having two columns one with udt array type and other with basic natie types. It should still be reported

Reply:
UDT array and native type is covered in case-1 below

Do you mean native array datatype?
If so, i can just modify the case-1 here to have that too

Reply:
yes